modules/local/gff_summary.nf

```diff
@@ -14,21 +14,23 @@ process GFF_SUMMARY {
     path "versions.yml"              , emit: versions
 
     script:
-    """
-    process_dbcan_cazys \\
-        -hmm ${dbcan_hmm} \\
-        -ov ${dbcan_overview} \\
-        -g ${cds} \\
-        -v ${params.dbcan_database_version} \\
-        -o ${meta.id}_dbcan_cazys.gff
+    // TODO: re-enable dbcan when the module is completed
 
+    // process_dbcan_cazys \\
+    //     -hmm ${dbcan_hmm} \\
+    //     -g ${cds} \\
+    //     -ov ${dbcan_overview} \\
+    //     -v ${params.dbcan_database_version} \\
+    //     -o ${meta.id}_dbcan_cazys.gff
+
+    //        --dbcan-cazys ${meta.id}_dbcan_cazys.gff \\
```

Comment:
I have a feeling that argument should in in `modules.config`?

Reply:
I disabled those lines because the current version of dbCAN and the toolkit are incompatible.